improver_tests/utilities/test_mathematical_operations.py

```diff
@@ -494,5 +494,48 @@ def test_dimension_preservation(self):
         self.assertListEqual(result_coord_order, expected_coord_order)
 
 
+class Test_fast_linear_fit(IrisTest):
+    """Test the fast_linear_fit method"""
+
+    def setUp(self):
+        """Creates some random data to represent x and y."""
+        array_size = 25
+        self.x = np.random.random(array_size)
+        self.y = np.random.random(array_size)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Attribute name "y" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5319684)